classes/data-stores/ActionScheduler_DBStore.php

```diff
@@ -711,7 +711,7 @@ protected function bulk_cancel_actions( $query_args ) {
 
 		while ( $action_ids ) {
 			$action_ids = $this->query_actions( $query_args );
-			if ( empty( $action_ids ) ) {
+			if ( empty( $action_ids ) || ! is_countable( $action_ids ) ) {
```

Comment:
I don't think `query_actions()` can return anything other than an array. Is this check really needed here?